### ISSUE TITLE
Consider merging efforts with jorhett/puppet-mcollective?

### ISSUE THREAD

**Issue by jorhett** [voxpupuli/puppet-mcollective#279]
I am wondering if you'd be interested in merging efforts with me and the mcollective module I'm maintaining.

Reasons:
  + more eyes, more people better
  + supports Puppet 4 and IPv6 and I can help maintain that
  + update to Learning MCollective book is being worked on, and I could highlight this project instead of my own repo

I'm not intending to toss it at you and walk away. I'm talking about an active integration with a major version number change, after which point I'll help maintain it with you.

**Comment by igalic** [comment]
:+1:

**Comment by jhg03a** [comment]
It would also add support for the sshkey security provider.

**Comment by jyaworski** [comment]
I'm a fan. I'll volunteer to actively work with you on the merge. Please open a PR.

**Comment by jyaworski** [comment]
@jhg03a status?

**Comment by jorhett** [comment]
@igalic I'm sorry I got the invite from you to join, but then got copyedits back from O'Reilly and have been buried. Can you resend?

I am also thinking that perhaps we should work on the merged module as voxpopuli/mcollective so that we can eventually clear up that source of confusion. Thoughts?

**Comment by igalic** [comment]
:+1:

**Comment by jaxxstorm** [comment]
Is there anything I can do to help with this? I'd rather not switch mcollective modules at this point, but we have a big puppet 4 upgrade coming and currently we're blocked by this single module, which is a shame.

@jorhett anything needed?

**Comment by jyaworski** [comment]
Indeed, ping @jorhett. I'm about to update to puppet 4 in my environment.

**Comment by jorhett** [comment]
Sorry, it's just been time. I ended up being tied up with LP4 book until March 19th and then had a family emergency pull me out of town. I just got back and caught up with some crisis and home to start work on it this week.

My module does work with Puppet4 today if you're in need right away. I'm going to provide a transition path forward for users of both modules to the combined one.

**Comment by jaxxstorm** [comment]
Thanks! If there's anything we can do help, let me know.
